docket.spec

```diff
@@ -112,6 +110,38 @@ exit 0
 %doc contrib/docket_lighttpd_vhost.conf
 
 %changelog
+* Mon Jan 08 2018 Jeffrey Kwasha <JeffKwasha@users.noreply.github.com> 0.2.1-1
```

Comment:
Fair enough. Let's boil these changelogs down to 2-3 high-level items. We can put the details in the description of the new tag.

Reply:
Please feel free to change anything at all in docket.spec .
 I will try to make all the needed corrections, but please assume I will miss things.